packages/react-components/src/overlay/src/useRestoreFocus.ts

```diff
@@ -49,7 +53,7 @@ export function useRestoreFocus(scope, { isDisabled } = {}) {
                     } else {
                         // If there is no next element, blur the focused element to move focus to the element to restore.
                         // focusedElement.blur();
-                        elementToRestore.focus();
+                        (elementToRestore as HTMLElement).focus();
```

Comment:
As you know this code have been strongly inspired by spectrum and they don't have to cast to `HtmlElement` to use `.focus`.

Why do we?

![image](https://user-images.githubusercontent.com/794579/113328320-24a92700-92ea-11eb-97e5-280aed4821bf.png)

https://github.com/adobe/react-spectrum/blob/c2c187606d447a6daa185e0b0507c22883ab3147/packages/%40react-aria/focus/src/FocusScope.tsx#L324

Reply:
fixed, thanks